pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -1761,22 +1761,26 @@ public class Foo {
         </example>
     </rule>
 
-    <rule name="UnnecessaryLocalBeforeReturn"
+    <rule name="UnnecessaryLocalBeforeBranch"
```

Comment:
See my other comment about renaming rules or deprecating the rule and adding a new rule... By this change, you essentially removed the rule "UnnecessaryLocalBeforeReturn", which would break custom rulesets which reference this rule...